data/mods/Aftershock/items/ammo/10mm.json

```
@@ -0,0 +1,44 @@
+[
+  {
+    "id": "afs_10mm_caseless_FMJ",
+    "type": "AMMO",
+    "name": { "str_sp": "10mm FMJ caseless" },
+    "description": "10mm ammunition with a brass jacketed bullet.  For as long as you can remember bullets like thse have been used for military, law enforcement, and civilian use.  Being caseless, these cannot be disassembled or reloaded.",
+    "weight": "9 g",
+    "volume": "117 ml",
+    "price": 400,
+    "price_postapoc": 800,
+    "material": [ "brass", "powder" ],
```

```suggestion
    "material": [ "brass" ],
```
We can ignore whatever combustible adhesive we are using in place of powder for now?  Or we can create the material and allow it to be scraped off and turned into makeshift explosives?

---

Probably best to ignore it for now; and in the future maybe allow it to be scraped off?